src/js/hmm/querys.ts

```diff
@@ -23,3 +25,26 @@ export function useListHmms(page: number, per_page: number, term?: string) {
         keepPreviousData: true,
     });
 }
+
+type Error = {
+    response: {
+        status: number;
+    };
+};
```

Comment:
```suggestion
type ErrorResponse = {
    response: {
        status: number;
    };
};
```

I would be tempted to fill this out a little more with anything else that came in the response and bump it into `js/types/ts`